db/db.js

```diff
@@ -164,12 +164,12 @@ async function getHistoryOfSessions(locationid, clientParam) {
   }
 }
 
-async function getAllSessionsFromLocation(location, clientParam) {
+async function getAllSessionsFromLocation(locationid, clientParam) {
   try {
     const results = await runQuery(
       'getAllSessionsFromLocation',
       'SELECT * FROM sessions WHERE locationid = $1 order by start_time desc',
-      [location],
+      [locationid],
```

Comment:
Changed this variable name to make it clearer that what we're passing is just the Location's ID, not the whole Location object.